packages/amplify-cli/src/extensions/amplify-helpers/get-category-pluginInfo.ts

```diff
@@ -1,9 +1,7 @@
 export function getCategoryPluginInfo(context, category, service?) {
   let categoryPluginInfo;
 
-  const pluginInfosForCategory = context.pluginPlatform.plugins[category].filter(pluginInfo => {
-    return pluginInfo.manifest.type === 'category';
-  });
+  const pluginInfosForCategory = context.pluginPlatform.plugins[category];
```

Comment:
we needed to remove it so that we could call generateModels from the codegen package in other flows without creating plugin interdependencies. Alternatively we could update the filter to check for type === category OR util